server/src/engineModel/EngineModel.ts

```diff
@@ -100,6 +110,17 @@ export class EngineModel {
         this.completionProvider.addCompletionItem(ymlClass);
     }
 
+    private buildEnumValue(instanceXmlElement: any, sourceType: string): YmlObjectInstance {
+        const identifier = `${sourceType}::${instanceXmlElement.$.ident}`;
+        const instance = new YmlObjectInstance(identifier, this.uri);
+        instance.detail = `${identifier} − Value of enum ${sourceType}`;
```

Comment:
Any way to get rid of the `Value of enum` (while keeping `${identifier} − ${sourceType}`) in favor of an icon change in the suggestion list or something? Dunno which possibilities are made available to you here.

Reply:
CompletionItemKind has Enum and EnumMember as possible values. I'll try to use them.